triage/update_summaries.sh

```diff
@@ -26,8 +26,8 @@ fi
 date
 # cat << '#'
 table_mtime=$(bq --format=json show 'k8s-gubernator:build.week' | jq -r '(.lastModifiedTime|tonumber)/1000|floor' )
-if [[ ! -e triage_builds.json ]] || [ $(stat -c%Y triage_builds.json) -lt $table_mtime ]; then
-  echo "UPDATING" $table_mtime
+if [[ ! -e triage_builds.json ]] || [[ $(stat -c%Y triage_builds.json) -lt $table_mtime ]]; then
```

Comment:
When using `[[` you should put the `||` inside the `[[` and use one `[[` for the `if`